frontend/common/src/helpers/util.ts

```diff
@@ -105,3 +105,16 @@ export const emptyToNull = (s: InputMaybe<string>): string | null =>
  */
 export const emptyToUndefined = (s: InputMaybe<string>): string | undefined =>
   empty(s) || s === "" ? undefined : s;
+
+/**
+ * Accepts an array of arrays and flattens it (T[][] -> T[])
+ * @param arrArr T[][]  An array of arrays
+ * @returns T[]         Flattened version of the array
+ */
+export function flatten<T>(arr: T[][]): T[] {
```

Comment:
Is this different from [flatmap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) ?

Reply:
I guess it's a little more specific and convenient.

Reply:
tbh, I never knew that existed and really wish it showed up when I was researching 😆 . I don't think this helper is unique enough to make sense. Especially since right now we are only using it once right now.

Updated in [remove flatten in favour of flatMap](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3695/commits/9766eb31294a96683bfcd418bf785b9064f4918e)